cves/CVE-2013-4316.yml

```diff
@@ -103,9 +109,14 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer:
-  code:
-  fix:
+  answer: |
+    Automated unit tests were involved since the issue was first introduced. Tests ensured
+    that the feature functioned as planned. This issue would not have been caught by the
+    tests as it was an issue with default configuration. The default configuration enabled
+    an insecure option that opened up potential vectors. At the time, they did not realize
+    that this option have as large of a potential impact.
```

Comment:
Check the grammar in the last sentence. Probably a typo - but it should probably say "they did not realize that this option _would_ have as large of a potential impact"

Reply:
Fixed the grammar, thanks!